x/oracle/types/errors.go

```diff
@@ -8,7 +8,8 @@ import (
 
 // x/oracle module sentinel errors
 var (
-	ErrSample               = sdkerrors.Register(ModuleName, 1100, "sample error")
-	ErrInvalidMsg           = sdkerrors.Register(ModuleName, 1, "invalid input create price")
-	ErrPriceProposalIgnored = sdkerrors.Register(ModuleName, 2, "price proposal ignored")
+	ErrSample                     = sdkerrors.Register(ModuleName, 1100, "sample error")
```

Comment:
Please change the error code to zero.

Reply:
updated

Reply:
In the Cosmos sdk error documentation, it is advised that error codes start from, and include 2.

> The only restrictions on error codes are the following:
> - Must be greater than one, as a code value of one is reserved for internal errors.
> - Must be unique within the module.

https://docs.cosmos.network/main/build/building-modules/errors

EVMOS does this by using `iota + 2` as the starting var.